src/components/PlaylistTableCell.tsx

```diff
@@ -1,42 +1,86 @@
 import React from 'react'
-import { StyleSheet } from 'react-native'
+import { StyleSheet, TouchableWithoutFeedback } from 'react-native'
 import { PV } from '../resources'
 import { Text, View } from './'
 
 type Props = {
-  itemsTotal?: number
+  createdBy?: string
+  itemCount?: number
+  onPress?: any
   title?: string
 }
 
 export const PlaylistTableCell = (props: Props) => {
-  const { itemsTotal = 0, title } = props
+  const { createdBy, itemCount = 0, onPress, title = 'untitled playlist' } = props
+
+  const wrapperTopStyles = [styles.wrapperTop]
+  if (createdBy) wrapperTopStyles.push(styles.wrapperTopWithCreatedBy)
```

Comment:
does pushing here work since the array is a const?

Reply:
Yes. You can push to const arrays, and update const object fields.